BookingsApi/BookingsApi.DAL/Queries/GetBookingsByCaseTypesQuery.cs

```diff
@@ -65,6 +65,8 @@ public GetBookingsByCaseTypesQueryHandler(BookingsDbContext context, IFeatureTog
                 .Include(x => x.HearingType)
                 .Include(x => x.CaseType)
                 .Include(x => x.HearingVenue)
+                .Include(x=>x.Allocations)
```

Comment:
`.Include(x => x.Allocations).ThenInclude(x => x.JusticeUser)` would be preferable here as it's strongly typed

Reply:
done